buf.gen.yaml

```diff
@@ -1,22 +1,22 @@
 version: v1
 plugins:
-  - remote: buf.build/prost/plugins/prost:v0.2.1-1
+  - plugin: buf.build/community/neoeinstein-prost:v0.2.2
     out: proto/gen
     opt:
       - bytes=.
       - file_descriptor_set
       - compile_well_known_types
       - extern_path=.google.protobuf=::pbjson_types
-  - remote: buf.build/prost/plugins/serde:v0.2.1-1
+  - plugin: buf.build/community/neoeinstein-prost-serde:v0.2.3
```

Comment:
The [buf.build/community](https://buf.build/community) organization is managed by @bufbuild and the build source of these plugins is open-sourced at: github.com/bufbuild/plugins.